tensorflow_datasets/video/moving_mnist.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+import functools
+import numpy as np
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+from tensorflow_datasets.image.mnist import _MNIST_IMAGE_SIZE
+
+_OUT_RESOLUTION = (64, 64)
+_TOTAL_PADDING = tuple(o - _MNIST_IMAGE_SIZE for o in _OUT_RESOLUTION) # 36, 36
+_SEQUENCE_LENGTH = 20
+_IMAGES_PER_SEQUENCE = 2
+
+_citation = """
+@article{DBLP:journals/corr/SrivastavaMS15,
+  author    = {Nitish Srivastava and
+               Elman Mansimov and
+               Ruslan Salakhutdinov},
+  title     = {Unsupervised Learning of Video Representations using LSTMs},
+  journal   = {CoRR},
+  volume    = {abs/1502.04681},
+  year      = {2015},
+  url       = {http://arxiv.org/abs/1502.04681},
+  archivePrefix = {arXiv},
+  eprint    = {1502.04681},
+  timestamp = {Mon, 13 Aug 2018 16:47:05 +0200},
+  biburl    = {https://dblp.org/rec/bib/journals/corr/SrivastavaMS15},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+"""
+
+
+class MovingMnist(tfds.core.GeneratorBasedBuilder):
+
+  VERSION = tfds.core.Version("0.1.0")
+
+  def _info(self):
+    shape = (_SEQUENCE_LENGTH,) + _OUT_RESOLUTION + (1,)
+
+    # as Image - doesn't work with 1 as final dim?
+    # sequence = tfds.features.Image(shape=shape)
+
+    # as video - doesn't work with 1 as final dim?
+    # sequence = tfds.features.Video(shape=shape)
```

Comment:
Video should works.
tfds.features.Video(shape=(seq_lenqth, height, width, 1))

If not, this is a bug from our end. Which error are you seeing ?